src/pages/landing/index.tsx

```diff
@@ -1,8 +1,11 @@
 import Head from "next/head"
 import { styled } from "linaria/react"
-import FeelingLogo from "../../assets/icons/logo.svg"
-import TwitterSvg from "../../assets/icons/twitter.svg"
-import EmailSvg from "../../assets/icons/email.svg"
+
+import FeelingLogo from "../../assets/icons/logo.svg?sprite"
+import TwitterSvg from "../../assets/icons/twitter.svg?sprite"
+import EmailSvg from "../../assets/icons/email.svg?sprite"
+// @ts-ignore
+import IPhone from "../../../public/iphone_crop.png?resize&sizes[]=300&sizes[]=500&sizes[]=850"
```

Comment:
Not sure how to make typescript recognise this type - any suggestions?

Reply:
Seems like a common issue: https://github.com/cyrilwanner/next-optimized-images/issues/103
You think maybe in the type declaration if we did `*.png*` would that work?